configure.ac

```diff
@@ -3199,10 +3199,14 @@ AC_ARG_ENABLE(loadable-sqlite-extensions,
               AS_HELP_STRING([--enable-loadable-sqlite-extensions],
                              [support loadable extensions in _sqlite module, see Doc/library/sqlite3.rst (default is no)]),
               [],
-              [enable_loadable_sqlite_extensions="no"])
-
+              [enable_loadable_sqlite_extensions=no])
 AC_MSG_RESULT($enable_loadable_sqlite_extensions)
 
+AS_VAR_IF([enable_loadable_sqlite_extensions], [yes], [
+  AC_DEFINE(PY_SQLITE_ENABLE_LOADABLE_EXTENSION, 1,
+  [Define to 1 to build sqlite module with loadable extensions.])
```

Comment:
Nitpick:
```suggestion
  [Define to 1 to build the sqlite3 module with loadable extensions support.])
```

Reply:
BTW, are we using `AC_CONFIG_HEADERS` somewhere?

Reply:
yes, configure.ac has ``AC_CONFIG_HEADER(pyconfig.h)``

Reply:
Is that in another PR? I can't find it in this checkout.